draft-ietf-quic-transport.md

```diff
@@ -2497,16 +2496,34 @@ handshake is confirmed (see Section 4.1.2 of {{QUIC-TLS}}), an endpoint MUST
 send any CONNECTION_CLOSE frames in a 1-RTT packet.  However, prior to
 confirming the handshake, it is possible that more advanced packet protection
 keys are not available to the peer, so the frame MAY be replicated in a packet
```

Comment:
"replicated" is not accurate as it suggest the same error code can be used in both places.